test/unit/helpers/is-interactive-element-test.js

```diff
@@ -25,10 +25,17 @@ describe('isInteractiveElement', function () {
     });
   }
 
-  let nonInteractive = ['<a></a>', '<input type="hidden">', '<img>', '<div></div>'];
```

Comment:
`<a></a>` needs to come back as `nonInteractive`

Reply:
see previous comment as to why it was removed.